requirements.in

```diff
@@ -1,3 +1,3 @@
 -e file:.
 # DON'T FORGET TO UPDATE setup.py !!
```

Comment:
Do we need to do this?

Reply:
No haha, the requirement `>=52.9.0` is still valid for this release of dmcontent